cluster/gce/config-default.sh

```diff
@@ -105,3 +105,6 @@ ENABLE_CLUSTER_DNS=true
 DNS_SERVER_IP="10.0.0.10"
 DNS_DOMAIN="kubernetes.local"
 DNS_REPLICAS=1
+
+# Admission Controllers to invoke prior to persisting objects in cluster
+ADMISSION_CONTROL=NamespaceAutoProvision,LimitRanger,ResourceQuota
```

Comment:
It's easier to say what each does:

NamespaceAutoProvision ensures that if you create a resource in a namespace object that that namespace exists and if not, creates one.  There is other lifecycle work around namespaces ongoing in other in flight PRs.

LimitRanger checks incoming pods and sees if there is a LimitRange in that namespace and if so, it validates pod resource usage (cpu, etc) are within defined min max constraints. 

ResourceQuota enforces Hard usage limits in the namespace if that namespace has a quota applied.

Any miss to e2e was not intentional on my part.  I think e2e should run with the same options. 

Sent from my iPhone

> On Mar 11, 2015, at 10:02 PM, Zach Loafman <notifications@github.com> wrote:
> 
> In cluster/gce/config-default.sh:
> 
> > @@ -105,3 +105,6 @@ ENABLE_CLUSTER_DNS=true
> >  DNS_SERVER_IP="10.0.0.10"
> >  DNS_DOMAIN="kubernetes.local"
> >  DNS_REPLICAS=1
> > +
> > +# Admission Controllers to invoke prior to persisting objects in cluster
> > +ADMISSION_CONTROL=NamespaceAutoProvision,LimitRanger,ResourceQuota
> So what happens if these aren't defined? config-test.sh doesn't have them (which is what gce e2e uses), and I'm curious what values to use for GKE.
> 
> —
> Reply to this email directly or view it on GitHub.
>